fabfile/setup.py

```diff
@@ -35,6 +35,14 @@ def basic_testing():
     execute(common_setup, build_citus)
     execute(add_workers)
 
+@task
+@roles('master')
+def extension_testing(extension_install_tasks, extension_configure_task):
+    'Sets up Postgres + Citus cluster with the extensions in given order'
+    # extension testing supports runs only for single node setup
```

Comment:
We discussed that Onder. It is ok for our use case to run on single node. 

The problem when try to run on clusters with workers is that we run pg_regress with --load-extension flags to add other extensions during the actual extension's(extension to be tested) test and pg_regress first creates db and then creates extensions. It means dbs are not propagated because citus is loaded after the db creation. It can be done with multi workers but requires additional work (propagate dbs manually and run pg_regress manually which misses many important flag related to pg_regress in makefiles).